src/main/java/org/codehaus/mojo/versions/PluginUpdatesXmlRenderer.java

```diff
@@ -106,11 +105,8 @@ public void render()
 
     private static String getSummaryBlock( Map<Plugin, PluginUpdatesDetails> allUpdates )
     {
-        Collection<ArtifactVersions> allVersions = new ArrayList<ArtifactVersions>();
-        for ( PluginUpdatesDetails details : allUpdates.values() )
-        {
-            allVersions.add( details.getArtifactVersions() );
-        }
+        Collection<ArtifactVersions> allVersions = new ArrayList<>();
+        allUpdates.values().forEach( details -> allVersions.add( details.getArtifactVersions() ) );
```

Comment:
Streams will create that collection for you, similar to:

```suggestion
        Collection<ArtifactVersions> allVersions = allUpdates.values()
            .map( details -> details.getArtifactVersions() )
            .collect( toList() );
```

Please don't merge, it is missing a static import and the lambda can be replaced by a method reference.